pages/app-layout/skeleton.page.tsx

```diff
@@ -1,96 +0,0 @@
-// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
This dev page is gone, because we now can use all standard app layout dev pages to test the new version (using `?appLayoutWidget=true` query parameter)